### PR TITLE
WIP: updates to root readme

### PR DESCRIPTION
**Pull request by davidthor**
This PR includes changes to root README to reflect the newest usability of arcctl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,198 @@
...
+$ arcctl apply datacenter local https://raw.githubusercontent.com/architect-team/arcctl/main/examples/datacenters/local.yml
 ```
 
-## Cloud Providers
+Datacenters are designed to be a home for many environments, now and in the future. You can leave your datacenter running indefinitely to make it easier to create new test environments.
+
+#### 2. Create an environment
 
-Before you'll be able to interact with any cloud resources, you'll need to register your cloud credentials with arcctl:
+Once you've created a datacenter, you'll need to create an [environment](./src/environments) that you can deploy into. Environments are basically namespaces that can allow datacenters to power more than one environment (populate for non-production use cases). Go ahead and create one on the datacenter you just created:
```
Comment (mueschm) [inline README.md:82]: `populate` is this supposed to be `popular`?

File: README.md [hunk 2]
```diff
@@ -1,92 +1,198 @@
...
 
-Scroll down to see more information about how to register each provider. **We highly recommend taking the time to read through our Readme for your desired cloud provider.** While arcctl can help to simplify the managment of resources, each cloud provider has it's own quirks that can still add some complexity to getting started. Our step by step guide can help make sure everything wroks smoothly the first time.
+#### 3. Deploy
 
-### Supported providers
-- [x] [digitalocean](./src/%40providers/digitalocean/)
-- [x] [aws](./src/%40providers/aws/)
-- [ ] gcp
-- [ ] azure
+Finally we can test out deploying a component! We've curated a sample application designed to show off the power of dependencies, [architect-team/auth-example](https://github.com:architect-team/auth-example). This component is a full-stack web application that connects our [Ory Kratos component](https://github.com/architect-team/kratos-selfservice-ui-node) to handle user registration and login, which in turn connects to our [Mailslurper component](https://github.com/architect-team/mailslurper) to handle SMTP for email verification. The component architecture looks a bit like this:
```
Comment (mueschm) [inline README.md:90]: A plantuml diagram may be huge here to help showcase what is where and why. Especially if the developer has never heard of these before.  You kind of have it below but putting inline descriptions into the graph could be nice. Would also break up the wall of text.

Can help make it if you need.

File: README.md [hunk 3]
```diff
@@ -1,92 +1,198 @@
...
+  kratos-admin                                 service          architectio/kratos       local        create  complete  0s                                                  
+  frontend                                     service          architectio/kratos       local        create  complete  0s                                                  
+  kratos-public                                ingressRule      architectio/kratos       local        create  complete  0s    http://kratos-public.local.127.0.0.1.nip.io/  
+  auth                                         ingressRule      architectio/kratos       local        create  complete  0s    http://auth.local.127.0.0.1.nip.io/
+```
+
+#### Bonus: Dev environments
+
+Want to quickly test your components and application code? We've got you covered! Just run `arcctl up .` in the directory your component lives in andspecify a datacenter to power your environment. Architect will automatically create an environment, deploy the component to it in debug mode, and then stream the application logs to your terminal. Yes, this is just like docker-compose!
```
Comment (mueschm) [inline README.md:144]: `andspecify`

File: README.md [hunk 4]
```diff
@@ -1,92 +1,198 @@
...
+
+## For DevOps
+
+### Key features
+
+### Getting started
+
+#### 1. Create a few datacenters
+
+Every datacenter you create becomes someone that you or your team can create and run cloud environments on top of. We've created a few sample datacenters for you to try, but they all use our [datacenters framework](./src/datacenters/) which means you're able to tweak them or create entirely new ones.
```
Comment (mueschm) [inline README.md:158]: `Every datacenter you create becomes someone` -> `Every datacenter you create becomes somewhere`

File: README.md [hunk 5]
```diff
@@ -1,92 +1,198 @@
...
+
+```sh
+# A datacenter that runs components locally
+$ arcctl apply datacenter local https://raw.githubusercontent.com/architect-team/arcctl/main/examples/datacenters/local.yml
+
+# A datacenter that runs workloads on GCP Cloud Run
```
Comment (mueschm) [inline README.md:164]: Setting up GCP is complex, get why we are promoting this, but our getting started guide might want to link to the guide to setup the credentials here.

File: README.md [hunk 6]
```diff
@@ -1,92 +1,198 @@
...
+
+Want to quickly test your components and application code? We've got you covered! Just run `arcctl up .` in the directory your component lives in andspecify a datacenter to power your environment. Architect will automatically create an environment, deploy the component to it in debug mode, and then stream the application logs to your terminal. Yes, this is just like docker-compose!
+
+```sh
+$ arcctl up . --datacenter local
+```
+
+## For DevOps
+
+### Key features
```
Comment (ryan-cahill) [inline README.md:152]: Does devops not get any key features?

**Pull request merged**